src/neptune/api/searching_entries.py

```diff
@@ -88,12 +88,12 @@ def get_single_page(
     attributes_filter: Dict[str, Any],
     limit: int,
     offset: int,
-    sort_by: Optional[str] = None,
-    sort_by_column_type: Optional[SORT_BY_COLUMN_TYPE] = None,
-    ascending: bool = False,
-    types: Optional[Iterable[str]] = None,
-    query: Optional["NQLQuery"] = None,
-    searching_after: Optional[str] = None,
+    sort_by: Optional[str],
+    sort_by_column_type: Optional[SORT_BY_COLUMN_TYPE],
```

Comment:
Shouldn't it be just `SORT_BY_COLUMN_TYPE`? We always have a value for `sort_by` if I understand and there is always a provided/deducted type.

Reply:
Yes, that's correct. Done :wink: